File: SPA/src/app/models/ticket.model.ts
import { CostumerModel } from './costumer.model';
import { EventModel } from './event.model';

export class TicketModel {
  id: number;
  eventId: number;
  shippingDate: Date;
  saleDate: Date;
  costumerId: number;
  costumer: CostumerModel;
  event: EventModel;
}

File: Model/TicketModel.cs
using System;

namespace Thisney.Ticket.Model
{
    public class TicketModel
    {
        public int Id { get; set; }
        public int EventId { get; set; }
        public DateTime ShippingDate { get; set; }
        public DateTime? SaleDate { get; set; }
        public int? CostumerId { get; set; }
        public CostumerModel Costumer { get; set; }
        public EventModel Event { get; set; }
    }
}
